src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -3221,4 +3238,34 @@ protected static String getUrlPrefix(String resourceType) {
       return "urn:uuid:";
     }
   }
+
+  public static void lengthOfStay(org.hl7.fhir.r4.model.Encounter encounterResource, Encounter encounter) {
+    // Length of Stay 
+    Duration length_of_stay = new Duration();
+    // setValue gets inhereted from Quantity class
+
+    // Check if encounter.start or encounter.stop are null, and if encounter is finished
+    EncounterStatus status = encounterResource.getStatus();
+    Long longStart = encounter.start;
+    Long longStop = encounter.stop;
+
+    if(longStart != null && longStop != null && status == EncounterStatus.FINISHED) {
+      Date date_start = new Date(encounter.start);
+      Date date_stop = new Date(encounter.stop);
+      long diff = date_stop.getTime() - date_start.getTime();
+
+      long minutes = TimeUnit.MILLISECONDS.toMinutes(diff);
+
+      // Set information in the Duration object
+      length_of_stay.setCode("m")
+                    .setSystem("http://unitsofmeasure.org")
+                    .setValue(minutes);
+      
+      // Once the Duration object is constructed and defined, create and extension and add this information
+      String url = "https://verily-src.github.io/vhp-hds-vvs-fhir-ig/StructureDefinition/length-of-stay";
```

Comment:
Let's break this url down and parameterize them. 
```
final String BASE_VVS_EXTENSION_URL =
      "https://verily-src.github.io/vhp-hds-vvs-fhir-ig/StructureDefinition/";
final String LENGTH_OF_STAY_SUFFIX = "length-of-stay";

String losExtensionUrl = BASE_VVS_EXTENSION_URL + LENGTH_OF_STAY_SUFFIX;
```
we can maybe move the `final` strings as global variables in a constants class and access them where needed. or maybe just move them as class variables.